CMakeLists.txt

```diff
@@ -326,7 +326,12 @@ add_subdirectory(local_ads)
 if (PLATFORM_DESKTOP)
   add_subdirectory(openlr)
   add_subdirectory(generator)
-  add_subdirectory(skin_generator)
+  # TODO(rokuz): CMake can't create an executable with the same name as the name of
```

Comment:
base_tests are located in base so there is no clashing in binary dir. You have DIR/base/base_tests/ and DIR/base_tests (executable). In our case it tries to create the following structure: DIR/skin_generator/ and DIR/skin_generator(executable).